msticpy/context/vtlookupv3/vtlookupv3.py

```diff
@@ -386,7 +389,7 @@ async def _lookup_ioc_relationships_async(
         observable: str,
         vt_type: str,
         relationship: str,
-        limit: Optional[int] = None,
+        limit: int = None,
```

Comment:
Not clear why you're changing the type annotation here - if it's a nullable int it should be Optional[int]

Reply:
Acknowledged - Please see changes [in commit:817c6607afd96b6dee427e10c9c36bc733a13b77](https://github.com/microsoft/msticpy/pull/739/commits/817c6607afd96b6dee427e10c9c36bc733a13b77)